routes/posts-router.js

```diff
@@ -0,0 +1,167 @@
+// Libraries
+const router = require("express").Router();
+
+//Files
+const Posts = require("../data/db");
+
+// Create
+router.post("/", (req, res) => {
```

Comment:
i would recommend adding some comments for logic as well and not only separation